pyvista/core/composite.py

```diff
@@ -152,7 +153,7 @@ def bounds(self) -> Tuple[float, float, float, float, float, float]:
 
         """
         # apply reduction of min and max over each block
-        all_bounds = [block.bounds for block in self if block]
+        all_bounds = [cast(list, block.bounds) for block in self if block]
```

Comment:
Just need this IMO:
```suggestion
        all_bounds = [cast(list, block.bounds) for block in self if block]  # cast for mypy
```

Reply:
Yeah, thanks, I've already added a bit more verbose comment before each line (pushed it now). It should be trivial that the cast is there for the type checker, it might be better to add context about where the failure would be without it.

Reply:
Works for me. Abandon suggesion.